apps/datahub/src/app/search/search-summary/search-summary.component.html

```diff
@@ -2,6 +2,19 @@
   <gn-ui-results-hits></gn-ui-results-hits>
 </div>
 <div class="mt-4" *ngIf="publisher$ | async as publisher">
-  <div translate="">datahub.search.org</div>
-  <div class="font-title text-primary text-21">{{ publisher }}</div>
+  <div translate="">search.filters.active</div>
+  <div
+    class="inline-block bg-primary py-2 px-3 rounded font-medium text-gray-50 text-sm leading-none mt-2"
+  >
+    <div class="flex">
+      <div
+        class="close bg-white bg-opacity-50 text-primary w-4 cursor-pointer h-4 text-xl leading-3 hover:bg-opacity-100 rounded-full text-center transition-all"
+        style="line-height: 14px"
+        (click)="removeOrg()"
+      >
+        ×
+      </div>
+      <div class="publisher ml-3 mt-px flex-grow">{{ publisher }}</div>
```

Comment:
Do the `publisher` and `close` classes only serve in the tests as a target? If so, I'm wondering if there is a way to prevent adding the classes for the tests and still targeting the divs in a clean way.

Reply:
yes they are...
haven't hear about a better solution so far, what do you suggest ?